src/js/components/Drop/DropContainer.js

```diff
@@ -343,9 +367,21 @@ const DropContainer = forwardRef(
       dropOptions,
     ]);
 
+    // if restrictFocus is true place focus on the drop container
+    // if restrictFocus is firstElement put focus on first
+    // interactive element if none fall back to drop container
     useEffect(() => {
-      if (restrictFocus) {
+      if (restrictFocus === true) {
         dropRef.current.focus();
+      } else if (restrictFocus === 'firstElement') {
+        const firstFocusableElement = getFirstFocusableDescendantElement(
+          dropRef.current,
+        );
+        if (firstFocusableElement !== undefined) {
+          firstFocusableElement.focus();
+        } else {
+          dropRef.current.focus();
+        }
```

Comment:
Do we need this last else statement? It seems odd that it executes the same code as the first if statement

Reply:
I added it incase someone has `restrictFocus='firstElement'`  but they might not realize they don't have any interactive elements

Reply:
I can take out just put it there to be cautious

Reply:
```

  // if restrictFocus is true place focus on the drop container
    // if restrictFocus is firstElement put focus on first
    // interactive element if none fall back to drop container

```

Follow what I had commented

Reply:
Yeah I think we should take it out just to ensure we align to whatever the previous behavior is when restrictFocus is false or undefined.

Reply:
so it will never come into this whole block of code when restrictFocus is false just like before. 

I can take it out though.